extra/resources/controld

```diff
@@ -228,8 +228,12 @@ controld_monitor() {
     killall -0 ${OCF_RESKEY_daemon} >/dev/null 2>&1 ; rc=$?
 
     case $rc in
-      0) dlm_tool ls | grep -q "wait fencing"
-         if [ $? -eq 0 ]; then
+      0) smw=$(dlm_tool status -v | grep "stateful_merge_wait=" | cut -d= -f2)
+         if [ $smw -eq 1 ]; then
+             ocf_log err "DLM status is: stateful_merge_wait"
+             rc=$OCF_ERR_GENERIC
+         elif [ -z "$smw" ] && dlm_tool ls | grep -q "wait fencing" && \
+	      ! stonith_admin -H -V | grep -q "wishes to"; then
```

Comment:
I've added the ability to do stonith_admin -H '*' -V to get the history for all nodes.